Formula/harmony-one.rb

```diff
@@ -0,0 +1,62 @@
+class HarmonyOne < Formula
+  desc "CLI tool to interact with the Harmony Blockchain"
+  homepage "https://docs.harmony.one/"
+  url "https://github.com/harmony-one/harmony/archive/refs/tags/v4.2.1.tar.gz"
+  sha256 "d29609a99a3a2e9031d2a26414ffdc3e271c3bf9cf9331dfcb321715fef682d4"
+  license "LGPL-3.0-or-later"
+
+  depends_on "go" => :build
+
+  # arm64 issues: https://github.com/harmony-one/go-sdk/issues/271#issuecomment-932876822
+  depends_on arch: :x86_64
+  depends_on "gmp"
+  depends_on "openssl@1.1"
+
+  resource "go-sdk" do
```

Comment:
How are these resources updated? Please document this in comments above each resource.

Reply:
harmony repo is where everything happening, whenever we need to upgrade the resources, we should add the linked repo in the changelog. Besides that, I will update the harmony doc for how to install with brew and how to upgrade the formula after this is approved.